### PR TITLE
add a --custom arg to unify cli command

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47791)

## Description
Add a new test flag to the unify sdk command 

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

**Comment by omerKarkKatz** [comment]
I has 2 questions on design of this flag: 
1. The flag name - do you think its ok or have a better idea.  
2. Should the flag give the option to add in txt the label to append for example Beta, Test etc... although i think it will be more uniformed if it will always be the same label, that is send to the customers. of course can always be changed by hand in the yml before the unify. 
@yaakovi

File: demisto_sdk/commands/unify/integration_script_unifier.py [hunk 1]
```diff
@@ -163,12 +164,34 @@ def unify(self, file_name_suffix=None):
...
+        if self.test:
+            yml_unified = self.add_test(yml_unified)
+
         output_map = self.write_yaml_with_docker(yml_unified, self.yml_data, script_obj)
         unifier_outputs = list(output_map.keys()), self.yml_path, script_path, image_path, desc_path
         print_color(f'Created unified yml: {list(output_map.keys())}', LOG_COLORS.GREEN)
 
         return unifier_outputs[0]
 
+    def add_test(self, unified_yml):
```
Comment (esharf) [inline demisto_sdk/commands/unify/integration_script_unifier.py:176]: - Let’s change the name to something more informative (maybe something like `add_test_section`)
- please add type hints
- this method can be static
```suggestion
    @staticmethod
    def add_test(unified_yml: Dict) -> Dict:
```

File: demisto_sdk/commands/unify/tests/yml_unifier_test.py [hunk 2]
```diff
@@ -1040,3 +1040,83 @@ def test_add_contributors_support(tmp_path):
         contributor_url='',
     )
     assert unified_yml["display"] == 'Test Integration (Partner Contribution)'
+
+
+def test_add_test(tmp_path):
```
[the remark this answers is not in the document]
Comment (omerKarkKatz) [reply demisto_sdk/commands/unify/tests/yml_unifier_test.py:1045]: Done

File: demisto_sdk/commands/unify/integration_script_unifier.py [hunk 3]
```diff
@@ -163,12 +164,34 @@ def unify(self, file_name_suffix=None):
...
+                 if the fields exsits.
+        """
+        to_append = ' - Test'
+        if unified_yml['name']:
+            unified_yml['name'] += to_append
+        if unified_yml['commonfields']['id']:
+            unified_yml['commonfields']['id'] += to_append
+        if not self.is_script_package:
+            if unified_yml['display']:
+                unified_yml['display'] += to_append
```
[the remark this answers is not in the document]
Comment (omerKarkKatz) [reply demisto_sdk/commands/unify/integration_script_unifier.py:191]: Done

File: demisto_sdk/commands/unify/integration_script_unifier.py (hunk 1 above)
Comment (omerKarkKatz) [reply demisto_sdk/commands/unify/integration_script_unifier.py:176]: Done

File: demisto_sdk/commands/unify/tests/yml_unifier_test.py [hunk 4]
```diff
@@ -1040,3 +1040,98 @@ def test_add_contributors_support(tmp_path):
...
+
+    Then:
+        - Check that the function will not raise any errors.
+    """
+    unifier = IntegrationScriptUnifier(str(tmp_path))
+    unifier.add_test_section({})
+
+
+@pytest.mark.parametrize("flag", [True, False])
+def test_add_test_section_flag_integration(repo, flag):
```
Comment (yaakovi) [inline demisto_sdk/commands/unify/tests/yml_unifier_test.py:1084]: this test should be in the `integration_tests` folder

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: demisto_sdk/commands/unify/tests/yml_unifier_test.py (hunk 4 above)
Comment (omerKarkKatz) [reply demisto_sdk/commands/unify/tests/yml_unifier_test.py]: Done

**Review by esharf — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ the same as the last release note version.
 * Added support for new XSIAM entities in **create-content-artifacts** command.
 * Added support for Parsing/Modeling Rule content item in the **unify** command.
 * Added the integration name, the commands name and the script name to the known words by default when running the **doc-review** command.
-* Added a flag '-c' '--custom' to the **unify** command, if True will append to the unified yml name/display/id the label ' - Test'
+* Added a flag '-c' '--custom' to the **unify** command, if True will append to the unified yml name/display/id the custom label provided
```
Comment (esharf) [inline CHANGELOG.md:14]:
```suggestion
* Added an argument '-c' '--custom' to the **unify** command, if True will append to the unified yml name/display/id the custom label provided
```

File: demisto_sdk/commands/unify/integration_script_unifier.py [hunk 6]
```diff
@@ -148,24 +148,24 @@ def unify(self, file_name_suffix=None):
...
         """
             Args:
                 unified_yml - The unified_yml
             Returns:
-                 the unified yml with the id/name/display appended with the Test label
+                 the unified yml with the id/name/display appended with the custom label
                  if the fields exsits.
         """
-        to_append = ' - Test'
+        to_append = ' - ' + self.custom
```
[the remark this answers is not in the document]
Comment (omerKarkKatz) [reply demisto_sdk/commands/unify/integration_script_unifier.py]: Done

**Pull request merged**